docs/Guide.Contributing.md

```diff
@@ -120,13 +120,15 @@ launchctl setenv PATH $PATH
 ##### Changing Detox e2e test suite
 
 If you add, rename, or delete a test in `detox/test/e2e` suite, you also have to update Jest snapshots
-of expected artifacts. It is usually done in five steps:
-
-1. In `detox/test` project, run all end-to-end tests on iOS with `npm run e2e:ios-multi`.
-2. Update the snapshots with: `npm run verify-artifacts:ios -- -u`.
-3. Re-run Android tests with `npm run e2e:android`.
-4. Update the snapshots with: `npm run verify-artifacts:android -- -u`.
-5. Add the snapshots to your Git commit:
+of expected artifacts. It is usually done in seven steps:
+
+1. In `detox/test` project, build the ios project with `npm run build:ios`.
+2. Run all end-to-end tests on iOS with `npm run e2e:ios`.
+3. Update the snapshots with: `npm run verify-artifacts:ios -- -u`.
+4. In `detox/test` project, build the ios project with `npm run build:android`
```

Comment:
I think this line was added, did you mean to build the android project with this line?
```suggestion
4. In `detox/test` project, build the android project with `npm run build:android`
```